packages/config/config/devices/0x010f/fgwreu-111.json

```diff
@@ -16,132 +16,209 @@
 	"supportsZWavePlus": true,
 	"paramInformation": {
 		"11": {
-			"label": "LED frame colour when moving",
-			"description": "This parameter defines the LED colour when the motor is running.",
+			"label": "LED Indicator: Color When Moving",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 7,
-			"defaultValue": 1
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Diable",
+					"value": 0
+				},
+				{
+					"label": "White",
+					"value": 1
+				},
+				{
+					"label": "Red",
+					"value": 2
+				},
+				{
+					"label": "Green",
+					"value": 3
+				},
+				{
+					"label": "Blue",
+					"value": 4
+				},
+				{
+					"label": "Yellow",
+					"value": 5
+				},
+				{
+					"label": "Cyan",
+					"value": 6
+				},
+				{
+					"label": "Magenta",
+					"value": 7
+				}
+			]
 		},
 		"12": {
-			"label": "LED frame – colour when not moving",
-			"description": "This parameter defines the LED colour when the device motor is not running.",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 7,
+			"$import": "#paramInformation/11",
+			"label": "LED Indicator: Standby Color",
 			"defaultValue": 0
 		},
 		"13": {
-			"label": "LED frame – brightness",
-			"description": "This parameter allows to adjust the LED frame brightness.",
+			"label": "LED Indicator: Brightness",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 100,
-			"defaultValue": 100
+			"defaultValue": 100,
+			"unit": "%"
 		},
 		"24": {
-			"label": "Buttons orientation",
-			"description": "This parameter allows reversing the operation of the buttons.",
+			"label": "Invert Orientation",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Normal Orientation",
+					"value": 0
+				},
+				{
+					"label": "Inverted Orientation",
+					"value": 1
+				}
+			]
 		},
 		"25": {
-			"label": "Outputs orientation",
-			"description": "reversing the operation of Q1 and Q2",
+			"label": "Output Orientation",
 			"valueSize": 1,
 			"minValue": 0,
 			"maxValue": 1,
-			"defaultValue": 0
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Q1 up, Q2 down",
+					"value": 0
+				},
+				{
+					"label": "Q1 down, Q2 up",
+					"value": 1
+				}
+			]
 		},
 		"30": {
-			"label": "Alarm configuration - 1st slot",
+			"label": "Alarm Configuration: 1st Slot",
 			"description": "Which alarm frames and how the device should react",
 			"valueSize": 4,
 			"minValue": 0,
-			"maxValue": 255,
+			"maxValue": 4,
 			"defaultValue": 0
 		},
 		"31": {
 			"label": "Alarm configuration - 2nd slot",
 			"description": "which alarm frames and how the device should react",
 			"valueSize": 4,
-			"minValue": 0,
-			"maxValue": 255,
+			"minValue": 1,
+			"maxValue": 4,
```

Comment:
If there are only four options, can we define these as options so they use a drop-down? Do you know what they are?